### ISSUE TITLE
Lutris's handling of wineprefixes is not sufficient 

### ISSUE THREAD

**Issue by Kreyren** [lutris/lutris#1825]
Disclaimer: I didn't find any other issue in lutris's repo that is relevant and i've consulted this issue on discord with lutris's team on which i'm filing this issue.

Lutris removed League Of Legends, but is not aware of it -> meaning when new is about to be installer it tries to open non-existing wineprefix.
![image](https://user-images.githubusercontent.com/11302521/53683658-b1e0b280-3d03-11e9-8fee-e35462402e78.png)
![image](https://user-images.githubusercontent.com/11302521/53683713-5105aa00-3d04-11e9-9d31-02ff7c30ae9c.png)
note Games/league-of-legends present which is default for lol installer

### Steps to reproduce:
- Install any game from Lutris
- Remove it's wineprefix once installed

### Hotfix
Restart lutris

### Recommendation: 
Recommends lutris to read from `$HOME/Games` and list installers if expected wineprefix is present and do not list it if it's not. This check should be probably performed regularly by lutris to make sure that it lists only installers that are present and ready to be used.

in theory it would need lutris to prefix or suffix wineprefixes to differenciate them from user-created?

Lutris should also check if executable is present and prompt to remove wineprefix if not -> if wineprefix can't be used or is corrupted -> reinstall it

**Comment by tannisroot** [comment]
> Lutris should also check if executable is present 

it does and it errors out if it's not preset
Can you rephrase the issue and make it smaller? I really can't undestand the point of it

**Comment by Kreyren** [comment]
got banned again.. i tried, closing

**Comment by Kreyren** [comment]
Just to clarify: willing to resolve if such contribution is helpful
